apm-sync.sh

```diff
@@ -4,13 +4,34 @@ set -e
 set -u
 
 
+function is_darwin() {
+	if [ "$(uname | lower)" == 'darwin' ]; then
```

Comment:
I suppose 'lower' is your custom command.
Please use 'tr' for instance to ensure the portability.

```suggestion
	if [ "$(uname | tr '[:upper:]' '[:lower:]')" == 'darwin' ]; then
```

You can also just compare with a non-lowercased string (i.e., 'Darwin').

```suggestion
	if [ "$(uname)" == "Darwin" ]; then
```